README.rst

```diff
@@ -200,16 +200,10 @@ Megatron GPT training requires NVIDIA Apex to be installed.
 
 .. code-block:: bash
 
-    git clone https://github.com/NVIDIA/apex
+    git clone https://github.com/ericharper/apex.git
```

Comment:
@ericharper do we need to take care of this change? this apex seems hacked a little bit from its main.

Reply:
No this was done to make installation easier.